config/v1/types_console.go

```diff
@@ -35,3 +36,20 @@ type ConsoleList struct {
 	metav1.ListMeta `json:"metadata,omitempty"`
 	Items           []Console `json:"items"`
 }
+
+type Auth struct {
+	// configmap reference in the openshift-config namespace
+	// The console needs a CA bundle to validate the certificate used to
+	// talk to the OAuth server.
+	OAuthEndpointCAFile ConfigMapReference `json:"oauthEndpointCAFile"`
```

Comment:
@enj can you comment on if this is needed since it is only needed for keycloak?

Reply:
Discussion in office with @enj, we decided that no, we will not be supporting this at this time.